### PR TITLE
W-13143969 deployment options diagram gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/flex-architecture-basic-deployments.adoc [hunk 1]
```diff
@@ -4,31 +4,78 @@ include::_attributes.adoc[]
...
 
+== Routing Configuration Methods
+
+There are different routing configuration methods dependent on what mode Flex Gateway is running in or what technological stack Flex Gateway is deployed on.
+
+How connections between Flex Gateway and services are configured is independent of the deployment model.
+
+=== Control Plane
+
+In control plane routing configuration, Flex Gateway and service connections are configured in a remote control plane. For Flex Gateway, the control plane is Anypoint Platfrom. control plane configuration is only available in Flex Gateway Connected Mode.
```
Comment (seblaz) [inline gateway/1.5/modules/ROOT/pages/flex-architecture-basic-deployments.adoc]:
```suggestion
In control plane routing configuration, Flex Gateway and service connections are configured in a remote control plane. For Flex Gateway, the control plane is Anypoint Platfrom. Control plane configuration is only available in Flex Gateway Connected Mode.
```

File: gateway/1.5/modules/ROOT/pages/flex-architecture-basic-deployments.adoc [hunk 2]
```diff
@@ -4,31 +4,78 @@ include::_attributes.adoc[]
...
+
+=== Custom Resource Definitions (CRDs)
+
+In custom resource refinitions (CRDs) routing configuration, Flex Gateway and service connections are configured in custom resource yaml files.
```
Comment (seblaz) [inline gateway/1.5/modules/ROOT/pages/flex-architecture-basic-deployments.adoc]:
```suggestion
In custom resource definitions (CRDs) routing configuration, Flex Gateway and service connections are configured in custom resource yaml files.
```

File: gateway/1.5/modules/ROOT/pages/flex-architecture-basic-deployments.adoc [hunk 3]
```diff
@@ -4,31 +4,78 @@ include::_attributes.adoc[]
...
+
+For Flex Gateway, the CRDs are `APIinstance`, `Service`, `PolicyBinding`, and `Configuration` and are defined in the xref:flex-local-configuration-reference-guide.adoc[].
+
+In Flex Gateway CRDs define connections in Local Mode but are also necessary for some connections in Connected Mode.
+
+=== Kubernetes Ingress Resource
+
+Flex Gateways running in Local Mode and deployed on Kubernetes can use Kubernetes `Ingress` resources to define connection routing. When routing is defined via `Ingress` resources, Flex Gateway acts an Ingress Controller. `Ingress` resources are only avaliable in Kubernetes Local Mode.
```
Comment (seblaz) [inline gateway/1.5/modules/ROOT/pages/flex-architecture-basic-deployments.adoc]:
```suggestion
Flex Gateways running in Local Mode and deployed on Kubernetes can use Kubernetes `Ingress` resources to define connection routing. When routing is defined via `Ingress` resources, Flex Gateway acts as an Ingress Controller. `Ingress` resources are only avaliable in Kubernetes Local Mode.
```

**Review by seblaz — COMMENTED** [review]
Great work!

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**